tests/potential/test_potential.py

```diff
@@ -289,6 +289,21 @@ def test_save(self):
 
         temp.close()
 
+    def test_get_design_variables(self):
+        """Test _get_design_variables method."""
+        a = relentless.DesignVariable(value=1.0)
+        b = relentless.DesignVariable(value=2.0)
+        c = relentless.SameAs(a=b)
+        d = relentless.ArithmeticMean(a=b, b=c)
+
+        m = relentless.potential.PairParameters(types=('A',), params=('energy','mass','charge'))
+        m['A','A']['energy'] = 1.5
+        m['A','A']['mass'] = a
+        m['A','A']['charge'] = d
+
```

Comment:
Let's add one more simple test first: if `a` is just used for all 3 parameters (should return a 1 element tuple).